libs/blocks/merch/merch.js

```diff
@@ -302,11 +302,20 @@ async function openFragmentModal(path, getModal) {
   return modal;
 }
 
+function focusOnTab(url) {
```

Comment:
The naming seems a bit unintuitive to me, I was expecting an actual `.focus()` call, instead this returns a path containing some query parameters that are used at a later time to eventually focus a certain section. Maybe `setFocusableTab` or something else might be a better choice to make its purpose clear.